### ISSUE TITLE
Instructions for device setup deprecated

### ISSUE THREAD

**Issue by Tryfos** [make-all/tuya-local#296]
The documentation's instructions to retrieve the device ID and local key through Android point to a now deprecated setup:
https://github.com/codetheweb/tuyapi/blob/cdb4289/docs/SETUP_DEPRECATED.md#capture-https-traffic
Moreover the Jinvoo Smart App seems not to exist any more.
Should this be followed now? https://github.com/codetheweb/tuyapi/blob/cdb42893d4d74f5371e32fb7dc7267ca5fc5ede7/docs/SETUP.md
Or this? https://github.com/codetheweb/tuyapi/blob/master/docs/SETUP.md
Or could the addon implement the Tuya Cloud API account as rospogrigio/localtuya ?
In general what are the recommended instructions today to retrieve the device ID and local key?

**Comment by make-all** [comment]
Yes, those instructions are outdated.  The best way to get the key now is to sign up for a Tuya developer account, and get them from the portal (which is essentially what localtuya is doing, but it is using the API directly rather than asking the user to use the API Explorer on iot.tuya.com.

I will see if I can find some more up to date instructions, as those ones date back to the parent project (hence the reference to the Goldair app)

**Comment by Tryfos** [comment]
I'm browsing the API explorer now but I can't seem to find the local key, where is this located?

**Comment by make-all** [comment]
I found it under General Devices Management / Get Device Information, but I think there are multiple API functions that contain the local key.  Of course you need to get the device id first, but I think there is a location in the Tuya app where that is displayed, or you can find it using tuyapi/cli, tinytuya or other tools that can scan the local network for Tuya devices.

**Comment by Tryfos** [comment]
The device ID is right there, beside each linked device in Devices menu as far as I can remember (I'm not in front of my PC now).I think I browsed the general devices management menu and didn't find the local key, but I'll try again.
I think the tuya API account should be the recommended way today (if indeed the local key is accessible). I've tried the other suggestions in the past and, though successful in the end, they are rather cumbersome and hit or miss.
